test/core/xds/xds_client_test.cc

```diff
@@ -2792,6 +2798,47 @@ TEST_F(XdsClientTest, AdsReadWaitsForHandleRelease) {
   EXPECT_TRUE(stream->Orphaned());
 }
 
+TEST_F(XdsClientTest, Fallback) {
```

Comment:
I don't think this test belongs in this PR, since the actual fallback functionality isn't here yet.